pkg/volume/iscsi/iscsi_util.go

```diff
@@ -184,6 +195,17 @@ func (util *ISCSIUtil) DetachDisk(c iscsiDiskUnmounter, mntPath string) error {
 	return nil
 }
 
+func extractTransportname(ifaceOutput string) (iscsiTransport string) {
+	re := regexp.MustCompile(`iface.transport_name = (.*)\n`)
+	iscsiTransport = re.FindStringSubmatch(ifaceOutput)[1]
```

Comment:
in case iscsiadm doesn't produce the output as expected, checking for nil survives a panic.